io.github.bothlab.syntalos.yaml

```diff
@@ -0,0 +1,262 @@
+id: io.github.bothlab.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
```

Comment:
it's `fallback-x11` if wayland is supported

Reply:
The app currently works quite terribly on Wayland, as it relies on multiple windows being able to open (so the researcher can move for example the microscope display to another screen and place the control panel at a convenient spot). Think like GIMP with its original multi-window mode.
Wayland has no way for Syntalos to tell the WM where to place windows relative to each other, which results in them appearing in random places when a project is loaded (not great), and in addition to that, all windows have the same icons, making them a bit harder to distinguish.
So, while it is possible to run on Wayland, the experience is not that great yet...

Reply:
them disable wayland.

Reply:
That's an option... But that way, testing on Wayland will be much harder for people who *do* want to try it. In the long run we need some way to get this to play nice with Wayland anyway (maybe by addressing https://gitlab.freedesktop.org/wayland/wayland-protocols/-/issues/72 in some way that makes everyone happy).
I need to think about this some more...

Reply:
Flathub is meant to distribute apps that work. If wayland doesn't wayland don't activate wayland. Developers can still chose to enable it. But otherwise not. You don't even force the QPA to be xcb so basically the flatpak is configure to use a setting that doesn't work.

Reply:
Fair enough. Wayland does work, but the experience is not really great while using it. I can always enable it again once we have tested this properly with Wayland and maybe compositors support multi-window apps better.